### ISSUE TITLE
Feature request: iCal

### ISSUE THREAD

**Issue by zpartakov** [leibler/dokuwiki-plugin-todo#159]
Hi
it would be very useful to have a url with an iCal content for the tasks, so that it may be integrated with online calendars, with due_date and start_date
if it could be filtered by task owner it would be a plus
thanks for your great extension

**Comment by runout-at** [comment]
Can you give examples how this should work and how it should look like.

**Comment by zpartakov** [comment]
yes: a url (button or sthing else, the easier for you) download an iCal file with all the tasks with a due_date > TODAY

sthing like:

![ksnip_20230913-060427](https://github.com/leibler/dokuwiki-plugin-todo/assets/2698011/172ec5c5-4de3-40f6-9209-db39e4132136)
